cmd/root.go

```diff
@@ -55,6 +57,10 @@ func Execute() {
 func MakeHandler() http.Handler {
 	n := middlewares.GetApp()
 	r := router.Get()
+	// if auth is enabled
+	if config.PrestConf.AuthEnabled {
+		r.HandleFunc("/auth", controllers.Auth).Methods("POST")
```

Comment:
if jwt has assets we close the endpoint `/auth` being barred by 401 in middleware
```toml
[jwt]
default = true
```
```
❯ prest (avelino/auth) ✘ curl -i -X POST curl -X POST http://127.0.0.1:3000/auth -H "Content-Type: application/json" -d '{"email": "t@avelino.xxx", "password": "123456"}'
curl: (6) Could not resolve host: curl
HTTP/1.1 401 Unauthorized
Content-Type: application/json
X-Content-Type-Options: nosniff
Date: Sat, 31 Oct 2020 14:28:17 GMT
Content-Length: 54

{
	"error": "Required authorization token not found"
}%                                                                                                                                                                                                  ❯ prest (avelino/auth) ✘
```